packages/components/src/focal-point-picker/index.js

```diff
@@ -211,7 +211,7 @@ export class FocalPointPicker extends Component {
 				<div className="components-focal-point-picker_position-display-container">
 					<BaseControl label={ __( 'Horizontal Pos.' ) }>
 						<input
-							className="components-text-control__input"
+							className="components-range-control__number"
```

Comment:
We should never assume a class name of another component, as it will most certainly cause breakage in the future.

If it's meant to be a range control, it should be implemented as a `<RangeControl>` component.

Otherwise, its styles must be defined locally, and the class name assigned following the coding guidelines:

https://github.com/WordPress/gutenberg/blob/master/docs/contributors/coding-guidelines.md#naming